azurerm/resource_arm_managed_disk.go

```diff
@@ -77,10 +84,12 @@ func resourceArmManagedDisk() *schema.Resource {
 
 			"disk_size_gb": {
```

Comment:
Should we change how we get `disk_size_gb` now that it's not required? Swap to `GetOk`?

Reply:
we could - however given this is an Integer it wouldn't help (as the default value is still `0`) - as such I think the current `Get` with a check > 0 should work (from below):

```
if v := d.Get("disk_size_gb"); v != 0 {
```